src/xdocs/cmdline.xml.vm

```diff
@@ -168,27 +168,33 @@ java -D&lt;property&gt;=&lt;value&gt;  \
     <section name="Download and Run">
       <p>
         It is possible to run Checkstyle directly from the JAR file using
-        the <code>-jar</code> option. Download latest
+        the <code>-jar</code> option.
+      </p>
+      <p>
+        Follow the steps mentioned below to run Checkstyle locally:
+      </p>
+      <p>
+        1. Download latest
         <a href="https://github.com/checkstyle/checkstyle/releases/download/checkstyle-${projectVersion}/checkstyle-${projectVersion}-all.jar">
-        checkstyle-${projectVersion}-all.jar</a>.
-        An example of run would be:
+           checkstyle-${projectVersion}-all.jar</a>.
+           It is recommended to use configuration files that are embedded in jar files, but latest
```

Comment:
This isn't even a step 2, it is more a note which I think still makes sense being after the commands.

Reply:
See comment at https://github.com/checkstyle/checkstyle/pull/12378#issuecomment-1328271903, I have to agree here. It looks like we just enumerated the information that was already present, and moved some elements around.


I had hoped for some grammatical improvements and a bit of explanation about the `It is recommended to use configuration files that are embedded in jar files` part. It took me a while to realize that the user wasn't supposed to hack the jar file and extract the configs themselves.